projects/canopy/src/lib/filter-buttons/filter-button.interface.ts

```diff
@@ -0,0 +1 @@
+export type FilterVariant = 'select-one' | 'select-multiple';
```

Comment:
We have an ongoing discussion where we think these should be enums so that they can be referenced in code.
```ts
enum FilterVariant {
  SelectOne,
  SelectMultiple
}
```
Which means in the code we can do this instead of referring to the string value...
```ts
if (this.variant === FilterVariant.SelectMultiple) {
```
This comment is more for awareness, although I think this approach might be better, I'd rather we log it in an issue and then fix throughout the codebase, cc @aaronBery who I think has looked into it too)

Reply:
I personally don't agree with the enum but I think I wrote my reasons in an issue. I'll try to find it as this is not the first time I wrote about it ;) Definitely needs a decision but I think we can leave this as is for now.

Reply:
As far as I can remember @elenagarrone (and I may be wrong) you mentioned a speaker at one of the angular meet-ups was an advocate for types over enums but I don't think you managed to follow up on what the rationale was. If you can dig that out it would be amazing :-) the only downside of enums I can think of is that I imagine it adds to bundle size whereas types are just annotations. However there are some clear disadvantages too. In reviewing this PR I'm not using my IDE so values that have type security can look like magic strings when read on a GitHub diff which adds an overhead in validating that they are not. either way would be good to hear the arguments 👍

Reply:
Moving to [here](https://github.com/Legal-and-General/canopy/issues/25) for now just to ease the PR through